### ISSUE TITLE
Check Web Services

### ISSUE THREAD

**Issue by epontell** [phylotastic/phylotastic#15]

**Comment by arlin** [comment]
Enrico checking whether links to web services work.  Phylomatic API has changed.  Datelife example is not working.  Brian will handle.

**Comment by arlin** [comment]
We should have closed this earlier, as it just calls for checking the web services.  The result of the check was to identify two further issues.  The DateLife issue is on the list already.  I'll open a separate issue for fixing the Phylomatic issue

**Comment by epontell** [comment]
Not yet - we were finally able to run the phylomatic one and we need to update the description in the web page - hope to have it done this week

**Comment by arlin** [comment]
We need to process this ticket.  I checked all the web service demos at phylotastic.org.  
* web form still works
* examples of tnrastic still work
* datelife example doesn't work - I made a separate ticket for that
* phylomatic doesn't work, but I notice that the services.html file on github has been updated to fix this.  This change needs to be pushed to the web site.  
* TreeStore phylows query doesn't work - I hid this in the html.  Content just needs to be pushed to web server.  

We should put some examples of OpenTree queries on here.  I'll make a separate ticket for that. 

Closing this ticket
